components/tidb_query_normal_expr/src/builtin_math.rs

```diff
@@ -963,12 +963,6 @@ mod tests {
                 Datum::I64(-2),
                 Datum::I64(0),
             ),
-            (
```

Comment:
Why is this removed?

Reply:
Already have this case.

Reply:
Oh, sorry. I didn't find it's a duplicated case.